releng/org.eclipse.xtext.target/org.eclipse.xtext.target-oxygen.target

```diff
@@ -32,7 +32,7 @@
 		<unit id="org.eclipse.xtext.testlanguages.ide.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.tests" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.tests.source" version="0.0.0"/>
-		<repository location="https://ci.eclipse.org/xtext/job/xtext-core/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+		<repository location="https://ci.eclipse.org/xtext/job/xtext-core/job/PR-1540/1/artifact/build/p2-repository/"/>
```

Comment:
wonder if you sould reference a newer one here
https://ci.eclipse.org/xtext/job/xtext-core/view/change-requests/job/PR-1540/lastSuccessfulBuild/artifact/build/p2-repository/

Reply:
Updated the repository location to the above.